test/ProdTest.js

```diff
@@ -17,10 +15,11 @@ const multiWrapper = '0x931e32b6d112f7be74b16f7fbc77d491b30fe18c';
 
 describe.skip('ProdTest', async function () {
     before(async function () {
-        // this.compoundWrapper = await CompoundWrapper.new();
+        // this.compoundWrapper = await CompoundWrapper.deploy();
```

Comment:
I think it will be useful to add `await this.compoundWrapper.deployed();` to the end of this comments